File: PHP/account.php
<?php
class Account{
  public $id;
  Public $name;
  Public $document;
  Public $email;
  Public $password;
  public function __construct($name, $document, $email, $password )
  {
    $this->name = $name;
    $this->document = $document;
    $this->email = $email;
    $this->password = $password;
  }
  public function printDataAccount(){
    echo "<br>Name: {$this->name} <br>Document: {$this->document} <br>Email: {$this->email} <br>Password {$this->password}";
  }  
}

File: Java/Account.java
class Account {
  Integer id;
  String name;
  String document;
  String email;
  String password;
  public Account(String name, String document, String email, String password){
    this.name = name;
    this.document = document;
    this.email = email;
    this.password = password;
  }
    void printDataAccount(){
      System.out.println("\tName: " + this.name +"\n\tDocument: " + this.document + "\n\tEmail:" + this.email +"\n\tPassword: " + this.password);
  }
}